### PR TITLE
Notify user of pod action failure

### PR DESCRIPTION
**Pull request by justinvdm**
We need a way to notify the user when pod actions fail.

### CODE REVIEW

**Comment by justinvdm** [comment]
@rudigiesler @KaitCrawford ready for review
@rowanseymour ping to keep you in the loop

File: templates/cases/case_read.haml [hunk 1]
```diff
@@ -66,6 +66,8 @@
 
     .row
       .col-md-8
+        <cp-notifications></cp-notifications>
```
Comment (rowanseymour) [inline templates/cases/case_read.haml]: Bonus points to anyone who wants to replace all the `UtilsService.displayAlert(..)` calls with something like this

**Comment by justinvdm** [comment]
@rudigiesler ready for review again.

**Comment by rudigiesler** [comment]
:+1:

**Pull request merged**